### DISCUSSION TITLE
minimum widget sizes

### DISCUSSION THREAD

**Discussion by qnpfrbreak** [iSoron/uhabits#835] [Feature Requests]
I made a lot of 1 x 1 widgets including frequencies and checks. But I found that I can not make new 1 x 1 frequency widget anymore. (still can make 1 x 1 check widget) Would you change the minimum size of the widgets to 1 x 1?

**Comment by hiqua** [comment]
Can you include info such as versions (before / after) and which Android / phone you're using?

**Reply by qnpfrbreak** [reply]
Sorry for missing my phone info. I use Galaxy S10 5G (SM-G977N) In Korea. Android version is 11, One UI version is 3.1.
I updated my phone (One UI version 3.0 (not sure) -> 3.1) on 8th of March and I tried to use 1 x 1 widget on the same day. (After the update)

But I can not sure the latest update causes problem because I hadn't added a widget for a long time and I updated my phone 3~4 times during the period.

Please let me know if I missed info to write down.

**Comment by iSoron** [comment]
Thank you for letting us know. 1x1 frequency widgets were never allowed, but some launchers never enforced this restriction. It seems like the One UI launcher is now enforcing it.

When this restriction was introduced, phones were fairly small, so a 1x1 graph widget did not make sense; there was simply not enough space available to render all the elements. Now that phones are so large, perhaps it's time to completely remove it.

**Reply by qnpfrbreak** [reply]
Thank you for accepting feedback! I hope everything goes well with.

**Comment by SamodAas** [comment]
I've just ran into the same issue as qnpfrbreak and I can only agree that it would be very convenient if there was no size limit, which would allow us much more flexibility on the placement of our widgets. I used to have all habit checkers and best streak widgets right next to each other, all in one window.

**Comment by ezio-melotti** [comment]
I agree with removing this limitation, and created this issue:
* https://github.com/iSoron/uhabits/issues/2118
